.github/workflows/build.yml

```
@@ -173,6 +174,8 @@ jobs:
       ${{ matrix.platform }}-${{ matrix.architecture }}
     needs:
       - diagnostics
+    # This job does not need any permissions
+    permissions: {}
```

I'm not sure about the output from the monitor Action, but I believe it should have this permission.

```suggestion
    permissions:
      # actions/checkout needs this to fetch code
      contents: read
```

---

Please see commit [372b896](https://github.com/cisagov/skeleton-ansible-role/pull/206/commits/372b896de5c7206f95fc255d5c3a66f2fb9aca39).